js/damage.js

```diff
@@ -593,6 +598,10 @@ function buildDescription(description) {
     if (description.isHelpingHand) {
         output += "Helping Hand ";
     }
+    output += description.attackerName + " ";
```

Comment:
The `attackerName` was already added to the display four lines up, so this line is unnecessary. Just add an extra space in `" Defense Curl "` two lines down.